src/Preg.php

```diff
@@ -170,25 +170,27 @@ private static function removeUtf8Modifier($pattern)
     private static function newPregException(int $error, string $errorMsg, string $method, array $patterns): PregException
```

Comment:
I have the feeling that we could remove this method and use `ExecutorWithoutErrorHandler` directly in public methods, yet out of time on my side. Let's unblock PHPUnit v10 and hopefully we come back to refactor this Preg class afterwards.